lib/libc/misc/lib_envpath.c

```diff
@@ -138,7 +139,7 @@ ENVPATH_HANDLE envpath_init(FAR const char *name)
 		printf("Allocate a container failed\n");
 		return (ENVPATH_HANDLE) NULL;
 	}
-
+	DEBUG_SET_CALLER_ADDR(envpath);
```

Comment:
`lib_malloc' is a preprocessor directive. It is replaced with `malloc` or `kmm_malloc`.
So, in this case we are not actually using a wrapper API. So, do we actually need DEBUG_SET_CALLER_ADDR in this case?

Reply:
The main purpose of saving the owner is getting debug point of memory issue like memory leak or corruption. But allocation in common codes like libc does not save real owner. We can't find who handles the allocation memory. This will resolve above. That's why we add this here.